api/v1alpha1/condition_consts.go

```diff
@@ -104,6 +104,9 @@ const (
 
 	// VirtualMachineImageProviderReadyCondition denotes readiness of the VirtualMachineImage provider.
 	VirtualMachineImageProviderReadyCondition ConditionType = "VirtualMachineImageProviderReady"
+
+	// VirtualMachineImageProviderSecurityComplianceCondition denotes security compliance of the library item provider.
+	VirtualMachineImageProviderSecurityComplianceCondition ConditionType = "VirtualMachineImageProviderSecurityCompliance"
```

Comment:
This was probably just when the v1a2 branch was first cut. I can sync these over later, since there are other image fields missing in v1a2 like `ContentVersion` that we'll require.